CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Major: Added customizable shortcuts. (#2340)
+- Minor: Refactor variable names to reduce build warnings (#3369)
```

Comment:
Changelog should be Dev: and in the dev section. I wish you luck on this PR 😁

Reply:
I believe it's not even needed at all, it's not relevant enough since it's just removal of some unused variables and variable name changes.

Reply:
Reverted changelog entry